gcovr/writer/txt.py

```diff
@@ -21,26 +21,30 @@
     presentable_filename,
     open_text_for_writing,
 )
-from ..coverage import CovData, calculate_coverage
+from ..coverage import CovData, CoverageStat, FileCoverage
 
 
 def print_text_report(covdata: CovData, output_file, options):
     """produce the classic gcovr text report"""
 
     with open_text_for_writing(output_file, "coverage.txt") as fh:
-        total_lines = 0
-        total_covered = 0
-
         # Header
         fh.write("-" * 78 + "\n")
         fh.write(" " * 27 + "GCC Code Coverage Report\n")
         fh.write("Directory: " + options.root + "\n")
 
         fh.write("-" * 78 + "\n")
-        a = options.show_branch and "Branches" or "Lines"
-        b = options.show_branch and "Taken" or "Exec"
-        c = "Missing"
-        fh.write("File".ljust(40) + a.rjust(8) + b.rjust(8) + "  Cover   " + c + "\n")
+        title_total = "Branches" if options.show_branch else "Lines"
+        title_covered = "Taken" if options.show_branch else "Exec"
+        title_missing = "Missing"
+        fh.write(
+            "File".ljust(40)
+            + title_total.rjust(8)
+            + title_covered.rjust(8)
```

Comment:
Create global constants for column length and use them also in the line for the files.